lib/models/city.dart

```diff
@@ -0,0 +1,9 @@
+class City {
+  String sehirAdi;
+  String sehirId;
+
+  City({
+    this.sehirAdi,
+    this.sehirId,
+  });
+}
```

Comment:
You can rename this variables in English. Ex; cityName, cityID

Reply:
@mukireus already had them named in English so i just moved them, but i think it's a good idea to rename them.